register/index.html

```diff
@@ -16,7 +16,11 @@
 </head>
 
 <body>
-
+  .tetris
+ .block1
+ .block2
+ .block3
+ .block4
```

Comment:
If you are using https://codepen.io/JorenVanHee/pen/ycodJ?page=3& as a reference, use the compiled HTML, which you can find in dropdown of HTML section. Replace this code with the compiled one.